charts/pravega/README.md

```diff
@@ -25,14 +25,16 @@ $ helm install [RELEASE_NAME] pravega/pravega --version=[VERSION] --set zookeepe
 ```
 where:
```

Comment:
Newline needed between this line and the bullet list below in order for bullet points to render correctly.

Reply:
Fixed